modules/brainbrowser/jsx/Brainbrowser.js

```diff
@@ -0,0 +1,101 @@
+/* exported RBrainBrowser */
+
+/**
+ * Brainbrowser Page.
+ *
+ * Renders Brainbrowser main page
+ *
+ * @author Alex Ilea
+ * @version 1.0.0
+ *
+ * */
+var BrainBrowser = React.createClass({
+
+  getInitialState: function() {
+    return {
+      defaultPanelSize: 300
```

Comment:
No big deal but, I don't think this needs to be kept in the state. I could be set a property of Brainbrowser. 
ie. Brainbrowser.defaultPanelSize instead of Brainbrowser.state.defaultPanelSize.